src/examples/zap-baseline.py

```diff
@@ -349,6 +355,9 @@ def main(argv):
       logging.debug ('Ajax Spider complete')
 
     # Wait for passive scanning to complete
+    while (datetime.now() - start).seconds < ((mins * 60) + 10):
```

Comment:
Is this check really needed? Is it to avoid polling the API? The script is already waiting for the passive scanner to finish (also, reusing the arg from the spider limit is not the best).